### PR TITLE
HashKey/RangeKey table support at upsert_with_expression mode

### PR DESCRIPTION
**Pull request by takasick**
I try to load HashKey/RangeKey defenishion table with upsert_with_expression mode.
but, it did not work.

so, I fix the code.

# example

## destination table
| uid(String,HK) | code(Number,RK) | count(Number) |
| --------------- |---------------:| -------------------- |
| u001 | 1 | 0 |
| u001 | 2 | 1 |
| u002 | 1 | 1 |

## load data
```
u001,1,1
u002,1,3
```

## update_expression
```
"ADD #count : count".
```

please merge this change

### CODE REVIEW

**Comment by sakama** [comment]
@takasick Thanks for your PR. I'll check in few days.

By the way, could you rewrite above your description in english when you can?
Writing in japanese may block non japanese contributors.
http://www.slideshare.net/tagomoris/how-to-createimprove-oss-products-and-its-community/13

**Comment by takasick** [comment]
@sakama Thank you for reply. and sorry, I rewriteed first comment.

File: src/main/java/org/embulk/output/dynamodb/DynamodbUtils.java [hunk 1]
```diff
@@ -178,11 +203,22 @@ protected String getPrimaryKeyName(DynamoDB dynamoDB, String tableName)
...
+            if (element.getKeyType().equals(KeyType.HASH.toString())) {
+                primaryKey = element.getAttributeName();
+            }
         }
         return primaryKey;
     }
 
+    protected List getPrimaryKey(DynamoDB dynamoDB, String tableName)
+    {
+        log.debug("getPrimaryKey() called");
```
Comment (sakama) [inline src/main/java/org/embulk/output/dynamodb/DynamodbUtils.java]: I think debug something value may OK but method call log isn't required.
What do you think about?

**Review by sakama — COMMENTED** [review]
@takasick Sorry for late.
I put 1 minor comment. Everything else looks good to me.
Could you take a look?

File: src/main/java/org/embulk/output/dynamodb/DynamodbUtils.java (hunk 1 above)
Comment (takasick) [reply src/main/java/org/embulk/output/dynamodb/DynamodbUtils.java]: I missed it. I also think that it is not necessary.
I deleted unnecessary log output and committed it.

**Comment by sakama** [comment]
Merged & released as [v0.1.4](https://rubygems.org/gems/embulk-output-dynamodb/versions/0.1.4).
Thank you for contributing 👍

**Comment by takasick** [comment]
:smile:

**Pull request merged**